packages/neon/neon/lib/src/router.dart

```diff
@@ -29,6 +31,15 @@ class AppRouter extends GoRouter {
           redirect: (final context, final state) {
             final account = accountsBloc.activeAccount.valueOrNull;
 
+            final loginQrcode = parseLoginQrcodeURL(state.location);
+            if (loginQrcode != null) {
+              return LoginQrcodeIntermediateRoute(
+                serverURL: loginQrcode.server,
+                loginName: loginQrcode.user,
+                password: loginQrcode.password,
+              ).location;
+            }
```

Comment:
I assume this is where the intet get's catched?

Reply:
Exactly. Sadly it doesn't contain the full URI so we have to do this matching. The whole `nc://login` part is removed